scripts/pr_check.py

```diff
@@ -0,0 +1,36 @@
+'''
+A specified script to check plugins on pull requests.
+'''
+
+import json
+import os
+
+from constants import ROOT
+from main import check
+
+with open(os.path.join(ROOT, '.github/outputs/all_changed_and_modified_files.json'), 'r', encoding='utf8') as f:
+	modified_files = json.load(f)
+
+folders = []
+
+for f in modified_files:
+	folder_name = os.path.split(os.path.dirname(f))[-1]
+	if folder_name not in folders:
+		folders.append(folder_name)
```

Comment:
There is no check to confirm whether the folder name truly corresponds to a plugin id, which might lead to a hidden issue.

Please ensure that `f` is indeed a file located within the `./plugins/<plugin_id>/` directory.